public/js/script.js

```diff
@@ -50,6 +73,10 @@ function loadHoods() {
   });
 }
 
+/**
+ * @function loadStops
+ * @description when invoked loadStops() loads stations from server http get requst and iterates through each station. If station doesnt exist we return out of the function otherwise grab the latitude and longtitude coordinates and paint the map accordingly. 
```

Comment:
This function doesn't make a request to the server, the data is bootstrapped before the function is invoked. I guess we could name `getStations` better since it's not a function and the data is already there.
Also the condition to return out the function if `station['Stop Name']` doesn't exist is for the forEach's callback function. It's doesn't exit out the `loadStops` function.

Reply:
@davis-trem 

Ah yes I think I follow, since the data is already provided via the CSV file and you are already "loading" that data up in the index.ejs file when the DOM / script tag is done scanning there is no need to do an HTTP request. 

I guess when I first read the loadStop function in the script.js file I assumed an HTTP request was made because the word "load" was in the naming of the loadStop function. So yes I think renaming the function to "getStations" is appropriate.  

So here are my proposed changes: 

@function - getStations <<<< (renaming the function) 
@description - gets Stations data from Stations .csv and iterates through each Station. If the Station doesnt exist it returns out the forEach's callback function otherwise grab the latitude and longtitude coordinates and paint the map accordingly via addTo(map) method.

Reply:
So that isn't what's happening either. The frontend doesn't handle the csv. The csv is read on the server side here: https://github.com/davis-trem/mtaHood/blob/bcbc2bf221b337466c68bf408dea9c68dd74f89f/index.js#L18. The server then bootstraps the station data to the frontend client. So the data is in the front as a global var. We name that var here (`getStations`): https://github.com/davis-trem/mtaHood/blob/bcbc2bf221b337466c68bf408dea9c68dd74f89f/views/index.ejs#L14. Then we use that global var to draw the stations on the map here: https://github.com/davis-trem/mtaHood/blob/bcbc2bf221b337466c68bf408dea9c68dd74f89f/public/js/script.js#L53. So I guess we should rename `loadStops` to `drawStops` and rename `getStations` to `STATIONS`

@getromandev

Reply:
@davis-trem thank you for clearing this up for me my friend I follow you now 🙌 . I updated the first line of the @description. 

I understand what is happening in the background now so I tried to articulate it in the description. Not sure how confident I feel confident about it. It can use another review 🙏. 

Here are the updated proposed changes: 

@function - `drawStops` <<<< (renamed the function)
@description - gets station data from a global variable and iterates through each station. (STATIONS is a global variable boot-strapped on the front-end that is in turn derived from a .CSV READ on the server-side). 

If the Station doesn't exist it returns out the forEach's callback function otherwise it grabs the latitude and longitude coordinates and paint the map accordingly via addTo(map) method.

Reply:
🔥🔥🔥